### PR TITLE
Auto-install blog post by guest author Ngan Pham

### PR DESCRIPTION
**Pull request by martinemde**
After a rave review of rubygems/rubygems#6561 by Ngan Pham, I asked if they were interested in writing a blog post and this is it!

Thanks Ngan!

### CODE REVIEW

**Review by ngan — COMMENTED** [review]
🙌 looks good!

File: _posts/2024-05-29-bundler-auto-install-just-got-a-whole-lot-better.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+```
+bundle config auto_install true
+# or
+bundle config --global auto_install true
+```
+
+Auto-install is not a new concept. Commands like `bundle (show | binstubs | outdated | exec | open | console | license | clean)` already benefited from the `auto_install` config.
+
+However, the [recent enhancement](https://github.com/rubygems/rubygems/pull/6561) by [@technicalpickles](https://github.com/technicalpickles) made it work with anything that uses `require "bundler/setup"`. This means even binstubs (which uses `require "bundler/setup"` by default) will now _just work_.
```
Comment (deivid-rodriguez) [inline _posts/2024-05-29-bundler-auto-install-just-got-a-whole-lot-better.md:28]:
```suggestion
However, the [recent enhancement](https://github.com/rubygems/rubygems/pull/6561) by [@technicalpickles](https://github.com/technicalpickles) made it work with anything that uses `require "bundler/setup"`. This means even binstubs (which use `require "bundler/setup"` by default) will now _just work_.
```

**Review by deivid-rodriguez — APPROVED** [review]
I suggested a grammar fix, but I'm not sure it's even correct 🤔.

Other than that, this is awesome! Thanks so much for telling your story and for the work and patience to land the improvement 😃

**Review by indirect — APPROVED** [review]

File: _posts/2024-05-29-bundler-auto-install-just-got-a-whole-lot-better.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Bundler Auto-Install Just Got A Whole Lot Better
+layout: post
+author: Ngan Pham
+author_email: nganpham@gmail.com
+---
+
+Working in a large monolith with many engineers, you never fail to get a flurry of changes everytime you pull from `main`.
```
Comment (simi) [inline _posts/2024-05-29-bundler-auto-install-just-got-a-whole-lot-better.md:8]: Would it make sense to make short note at the beginning this is host blog post by XY from Z company sharing their experience in italic or so?

**Pull request merged**